core/src/consensus.rs

```diff
@@ -67,12 +67,13 @@ impl Tower {
         bank_slot: u64,
         vote_accounts: F,
         ancestors: &HashMap<Slot, HashSet<u64>>,
```

Comment:
where is the dependency on ancestors?  if we remove this, then its guaranteed to be consistent.

Reply:
I think replay stage still relies on this set of ancestors being used in this computation in order to support `confirm_forks` which is how we do our confirmation times